edifikana/front-end/shared-compose/src/commonMain/composeResources/values/strings.xml

```diff
@@ -84,7 +82,14 @@
     <string name="title_timecard_view_staff">Empleado</string>
     <string name="title_timecard_staff_list">Lista de Empleados</string>
 
-    <string name="text_email">Correo/# Telefónico (requirido)</string>
-    <string name="text_password">Contraseña (requirido)</string>
```

Comment:
This is a clean-up of the name for the strings. With this change, all strings used in the "SignUp" screen will be prefixed with `sign_up_*`. This will help avoid collisions and conflicts with strings used across multiple screens.